packages/amplify-category-notifications/src/legacy-pinpoint-helper.ts

```diff
@@ -0,0 +1,280 @@
+    }
+  }
+
+  if (!pinpointApp) {
+    const scanOptions: $TSAny = {
+      isRegulatingResourceName: true,
+      envName: localEnvInfo.envName,
+    };
+
+    pinpointApp = scanCategoryMetaForPinpoint(amplifyMeta[AmplifyCategories.NOTIFICATIONS], scanOptions);
+    if (pinpointApp) {
+      resourceName = scanOptions.regulatedResourceName;
+    }
+  }
+
+  if (!pinpointApp) {
+    pinpointApp = scanCategoryMetaForPinpoint(amplifyMeta[AmplifyCategories.ANALYTICS]);
+    if (pinpointApp) {
+      resourceName = generateResourceName(pinpointApp.Name, localEnvInfo.envName);
+      constructResourceMeta(amplifyMeta, resourceName, pinpointApp);
+    }
+  }
+
+  if (!pinpointApp) {
+    printer.blankLine();
+    resourceName = await createPinpointApp(context, resourceName);
+  }
+
+  context.exeInfo.serviceMeta = context.exeInfo.amplifyMeta[AmplifyCategories.NOTIFICATIONS][resourceName];
+  context.exeInfo.pinpointApp = context.exeInfo.serviceMeta.output;
+};
+
+// resource name is consistent cross environments
+const generateResourceName = (pinpointAppName: string, envName: string):string => pinpointAppName.replace(getEnvTagPattern(envName), '');
+
+const generatePinpointAppName = (resourceName: string, envName: string):string => resourceName + getEnvTagPattern(envName);
+
+const getEnvTagPattern = (envName: string): string => (envName === 'NONE' ? '' : `-${envName}`);
+
+/**
+ * Legacy code to create pinpoint app using Pinpoint SDK
+ * @param context amplify cli context
+ * @param resourceName pinpoint app resource name
+ * @returns pinpoint app name
+ */
+const createPinpointApp = async (context: $TSContext, resourceName:string): Promise<string> => {
+  const { projectConfig, amplifyMeta, localEnvInfo } = context.exeInfo;
+
+  printer.info('An Amazon Pinpoint project will be created for notifications.');
+  if (!resourceName) {
+    resourceName = projectConfig.projectName + context.amplify.makeId(5);
+    if (!context.exeInfo.inputParams || !context.exeInfo.inputParams.yes) {
+      const answer = await inquirer.prompt({
+        name: 'resourceNameInput',
+        type: 'input',
+        message: 'Provide your pinpoint resource name: ',
+        default: resourceName,
+        validate: (name: $TSAny):$TSAny => {
+          let result = false;
+          let message = '';
+          if (name && name.length > 0) {
+            result = true;
+          } else {
+            message = 'Your pinpoint resource name can not be empty.';
+          }
+          return result || message;
+        },
+      });
+      resourceName = answer.resourceNameInput;
+    }
+  }
+
+  const pinpointAppName = generatePinpointAppName(resourceName, localEnvInfo.envName);
+  const pinpointApp = await createApp(context, pinpointAppName);
+  constructResourceMeta(amplifyMeta, resourceName, pinpointApp);
+  context.exeInfo.pinpointApp = pinpointApp; // needed for authHelper.ensureAuth(context);
+
+  printer.blankLine();
```

Comment:
why a blank line?

Reply:
removing as this method is no longer used